<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AntilogTestsUI</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../antilog/resources.qrc">
    <normaloff>:/artwork/artwork/icon.svg</normaloff>:/artwork/artwork/icon.svg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCheckBox" name="checkBoxFile">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>File</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxStdout">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>stdout</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxHelloWorld">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Hello world</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxExample1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Example1</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxPeriod">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>420</y>
      <width>121</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="value">
     <number>2000</number>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidgetPreview">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>170</y>
      <width>471</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditFile">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>441</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>430</y>
      <width>131</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Period [msecs]</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>440</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxSpdLog">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>88</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Spdlog</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxAppendSerial">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Append serial</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxSourcesAndModules">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Sources &amp;&amp; Modules</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDeleteFile">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxUDP">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>UDP</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditIP">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>113</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>localhost</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPort">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>80</y>
      <width>113</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>12345</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxMulticast">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Multicast</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditMulticastIP">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>113</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>239.255.43.21</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditMulticastPort">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>110</y>
      <width>113</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>12345</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>686</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../antilog/resources.qrc"/>
 </resources>
 <connections/>
</ui>
